src/main/java/lavalink/client/io/jda/JdaLavalink.java

```diff
@@ -67,17 +67,21 @@ public JdaLink getExistingLink(Guild guild) {
         return getExistingLink(guild.getId());
     }
 
+    /**
+     * May produce NullPointerException if JDA provider has not been initialised
```

Comment:
I've changed my mind. IllegalStateException is more appropriate.

Use `@throws` javadoc syntax